CHANGELOG.md

```diff
@@ -3,6 +3,22 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [[0.8.0](https://github.com/sanger-tol/blobtoolkit/releases/tag/0.8.0)] –  – [2025-05-02]
```

Comment:
@yumisims . Do you want to choose a pokemon name to name the release ?

Reply:
sprigatito?

Reply:
Oki !